src/pages/Aggregator/Aggregator.component.tsx

```diff
@@ -85,6 +97,12 @@ export const AggregatorComponent = ({
     setValue
   );
 
+  const slippageProtectionAvailable = startingChain === destinationChain;
+
+  useEffect(() => {
+    setValue(AggregatorInputs.ReceiveAmount, receiveAmount || '0.0');
```

Comment:
When a low amount of funds are converted, the penalty amount is not visible but I can see it applied to receive amount:
![image](https://github.com/BabelFishProtocol/babelfish-dapp/assets/765825/431b067f-a4b3-4035-a7d1-cd1af8a3b06d)